ex8_html-css-basics

Indigoinspn opened the pull request:
task-01.html Created an html page by given instructions.
task-02.html Created an html page by given layout.
task-03.html Several ways to create border with CSS.
task-04.html Applied different CSS styles.

Indigoinspn committed:
ex8_html-css-basics
task-01.html Created an html page by given instructions.
task-02.html Created an html page by given layout.
task-03.html Several ways to create border with CSS.
task-04.html Applied different CSS styles.

aneelia commented on src/ex8_html-css-basics/style.css:
```diff
@@ -0,0 +1,66 @@
+
+}
+
+#b-2 {            
```
исправь пожалуйста наименования b-1  и тд на понятные)

Indigoinspn replied:
Исправила

aneelia commented on src/ex8_html-css-basics/task-03.html:
```diff
@@ -0,0 +1,147 @@
+            border-color: rgb(204, 126, 235)
+        }
+
+        #b-2 {
```
здесь тоже

aneelia replied:
вообще смысл этог задания придумать 5 разных способов, а не вариаций использования border :) нужно переделать

Indigoinspn replied:
Переделала, посмотри пожалуйста)

aneelia commented:
и пустые строки в начале файлов убери

Indigoinspn committed:
Task-03 -  corrected id names and chande ways of creating element
border. Task-04 (style.css) - corrected id names and removed unnecessary empty lines.

aneelia commented on src/ex8_html-css-basics/task-04.html:
```diff
@@ -0,0 +1,86 @@
+        div[id*="b"] {
+            padding: 1%;
+            width: 30vw;
+            margin: 2% 5%;
```
margin: 2% 5%;
:(

Indigoinspn replied:
Заменила

Indigoinspn committed:
Change relative margin units to absolute.

Indigoinspn committed:
Change margin relative unites to absolute in task-03.html.
